res/values-in/strings.xml

```diff
@@ -84,13 +84,13 @@
     <string name="SIGN_OUT">"KELUAR"</string>
     <string name="ACHIEVEMENTS">"PRESTASI"</string>
     <string name="STATISTICS">"STATISTIK"</string>
-    <string name="XP_">"XP:"</string>
+    <string name="XP_">"Bahasa Indonesia: XP:"</string>
```

Comment:
Why did you write the language name here?

Reply:
I copied XP which came out in Indonesian

Reply:
Here it must keep as "XP" if there isn't any significant acronym translation in Indonesian